bin/selbot2.rb

```diff
@@ -218,6 +218,11 @@
       :expression => /:tias/,
       :text => "Try it and see - you'll be able to find out much faster if your proposal works by trying it than by asking us if it will work.",
       :help => "Try it and see"
+    },
+    {
+      :expression => /:path/,
+      :text .     => "https://seleniumhq.github.io/docs/wd.html#adding_executables_to_your_path",
+      :help .     => "Add WebDriver to your PATH"
```

Comment:
These `.` are not going to make it happy

Reply:
I assumed my screen was mucky, but apparently I just can't type :)